src/main/java/bookstore/dto/RegisterRequestDto.java

```diff
@@ -0,0 +1,23 @@
+package bookstore.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+public record RegisterRequestDto(
```

Comment:
in registration you had to make two fields password and password confirmation, and use your annotation to check that they match

Reply:
I know, but I actually don't see the point in doing so. Because it's redundant traffic on the server which in most cases will be validated in the client-side application.

Reply:
i think the point of this task is to understand how to create your own validation annotation